server/src/core/workspace/resolvers/workspace.resolver.ts

```diff
@@ -95,4 +98,22 @@ export class WorkspaceResolver {
 
     return paths[0];
   }
+
+  @UseGuards(AbilityGuard)
+  @CheckAbilities(DeleteWorkspaceAbilityHandler)
```

Comment:
Please make sure this policy check is working. Usually what I do is that I take the graphql http request sent by the front to the back in Chrome Dev Tools, export it as curl, import it in Postman and play with the data.
We should make sure that this is failing as "Unauthorized" when the user is not connected